docs/Deprecating-Disabling-and-Removing-Formulae.md

```diff
@@ -42,16 +44,17 @@ A formula should be disabled to indicate to users that the formula cannot be use
 
 The most common reasons for disabling a formula are:
 
-- it cannot be built from source (meaning no bottles can be built)
+- it cannot be built from source (meaning no new bottles can be built)
 - it has been deprecated for a long time
 - the upstream repository has been removed
 - the project has no license
```

Comment:
Not a suggestion to change, but just to discuss:

Do we still want to treat this differently to the "non-open-source license" case?

Reply:
I think so as there's often ambiguity here.